component/vshn_postgres.jsonnet

```diff
@@ -403,6 +413,8 @@ local sgCluster = {
 
     comp.FromCompositeFieldPath('spec.parameters.backup.schedule', 'spec.forProvider.manifest.spec.configurations.backups[0].cronSchedule'),
     comp.FromCompositeFieldPath('spec.parameters.backup.retention', 'spec.forProvider.manifest.spec.configurations.backups[0].retention'),
+    comp.FromCompositeFieldPath('spec.parameters.network.serviceType', 'spec.forProvider.manifest.spec.postgresServices.primary.type'),
```

Comment:
I just saw, that you also have logic in the composition function. Please do these patches there as well. It's confusing if the logic is spread over the composition function and the legacy P+T compostion.